remme/tp/account.py

```diff
@@ -97,7 +108,11 @@ def _transfer(self, context, public_key, transfer_payload):
         References:
             - https://github.com/Remmeauth/remme-core/blob/dev/remme/genesis/__main__.py
         """
-        address = self.make_address_from_data(public_key)
+        if transfer_payload.sender_account_type == TransferPayload.SenderAccountType.Value('ACCOUNT'):
```

Comment:
You can just use `TransferPayload.ACCOUNT` here.

Reply:
@eugene-babichenko, done.